external/anomaly/tests/anomaly_common.py

```diff
@@ -18,6 +18,10 @@
 from collections import namedtuple
 from copy import deepcopy
 from typing import List, Type
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from nncf.torch.nncf_network import NNCFNetwork
```

Comment:
And will you be able to move all these tests to somewhere in`tests` directory in the root `~/training_extensions/tests`?

Reply:
The idea is that we would like to combine all the tests to a single location, rather than scattering them around different places in the repo

Reply:
Same as above, thanks for addressing this.